data/unpack.go

```diff
@@ -0,0 +1,53 @@
+package data
+
+import (
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+// Unpack unpacks the assets from this package into a target directory.
+func Unpack(dir string) (err error) {
+	return unpack(dir, ".")
+}
+
+func unpack(base string, uri string) (err error) {
+	file, err := Assets.Open(uri)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	info, err := file.Stat()
+	if err != nil {
+		return err
+	}
+
+	if info.IsDir() {
+		os.Mkdir(base, 0777)
```

Comment:
does it have to be `0777`?

Reply:
> does it have to be 0777?

It doesn't *have* to be, but this is generic, and your `umask` will reduce it to whatever you feel is reasonable.